packages/calcite-components/src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -92,7 +94,7 @@ export class DropdownGroup {
       this.groupPosition > 0 ? <div class="dropdown-separator" role="separator" /> : null;
 
     return (
-      <Host aria-label={this.groupTitle} role="group">
+      <Host aria-label={this.groupTitle} id={this.el.id || this.guid} role="menu" tabindex="-1">
```

Comment:
Could you speak to the use case for changing the role from `"group"` to `"menu"`? Overall `"group"` provides more support for AT, in particular NVDA. This could be why NVDA isn't receiving context with the changes.

Role and AT support conveying the role and name:
- [Group role support](https://a11ysupport.io/tech/aria/group_role)
  - Supports role and name for JAWS, Narrator, NVDA and VoiceOver on desktop
- [Menu role support](https://a11ysupport.io/tech/aria/menu_role)
  - Supports role and name for JAWS and Narrator
  - Supports role only for NVDA and VoiceOver